lib/mrkt/concerns/crud_leads.rb

```diff
@@ -40,5 +40,17 @@ def associate_lead(id, cookie)
         json_payload(req, {})
       end
     end
+
+    def merge_leads(winning_lead_id, losing_lead_ids, merge_in_crm: false)
+      params = Faraday::Utils::ParamsHash.new
+
+      params[:leadIds] = losing_lead_ids.join(",") if losing_lead_ids
+      params[:mergeInCRM] = merge_in_crm
+
+      post("/rest/v1/leads/#{winning_lead_id}/merge.json?#{params.to_query}") do |req|
```

Comment:
![screen shot 2017-03-10 at 10 02 48 am](https://cloud.githubusercontent.com/assets/11512431/23770664/d5f14f14-0578-11e7-9cb2-fc6e4a764f86.png)

docs say query, tested it out in the body does work either